### PR TITLE
updating files to fix g11n scanner issues.

### PR DESCRIPTION
**Pull request by sivanatase1-ctxs**

### CODE REVIEW

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json [hunk 1]
```diff
@@ -1,173 +1,173 @@
...
+    "17c93a5e-45ee-4e5d-bdce-36389e931b1f": "旅程の詳細",
+    "1d6c4a67-7dc8-4407-919e-9ceb7c9d2e38": "プロジェクト名",
+    "e153d831-9222-425a-9838-735cbf7820cd": "帰り (日付)",
+    "1f7104e1-032f-42cb-b5c0-20dada2332e5": "予約者",
+    "3abc36b2-1d14-486b-a756-a9df558c8ae0": "旅程の共有",
+    "29e9a5d4-ebdb-4851-a10d-16386bb1cbf0": "予約方法",
+    "a6a1b0be-0831-4fe1-a667-fbf6bc1f2d07": "新しい旅程が予約されました",
+    "a41919dd-b075-416f-a748-40a6fea0ee46": "出張名",
+    "4f5ea46b-cf84-48a0-9fc5-328f74f970c8": "新しい共有旅程",
+    "c0bcc497-7d3a-448d-9c1c-adcbd5a045d7": "Itinerary Details",
```
Comment (ghost) [inline bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:16]: Why not translate this string?

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json [hunk 2]
```diff
@@ -1,173 +1,173 @@
...
+    "05f29eae-149e-4ee8-90cc-4fbf64d7cc75": "経費ポリシー",
+    "4facd8ae-fe98-439d-b704-a75a8cb01165": "経費",
+    "8ac505a2-84a9-4de5-ae3c-5d3a5d6c9f37": "{{format total \"decimal\"}} {{currency_code}}",
+    "dfdd1224-36e3-477e-8de7-91dcd3586357": "取引金額",
+    "c8a202f6-2903-48fc-ba88-3cdeec91ba5a": "表示するデータがありません",
+    "a6e03788-c4b7-4019-b714-700fce92e61d": "自分の経費",
+    "b5bb6768-e85a-4a3a-ac36-cb08e1f6e260": "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
+    "f70872db-3c1f-4e66-a929-f0e68684fcb2": "{{appr_login_id_report.last_name}} {{appr_login_id_report.first_name}}",
+    "dde0da2b-297f-4bd5-9252-6d909bd86777": "総額",
+    "ae4797ac-3815-4dfa-9041-3ff021e8a4e1": "Expense Details",
```
Comment (ghost) [inline bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:58]: Why not translate this string?

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json [hunk 3]
```diff
@@ -1,173 +1,173 @@
...
+    "ceddd6ca-18a0-477f-bd4d-7ef289f3d142": "支出カテゴリ",
+    "5dd37eb9-772a-46cf-aec9-f556fd9813a6": "自分の経費",
+    "6d0b9208-4963-4841-bb51-26d1aa9d302f": "Report Status",
+    "24f2ed54-c287-4e38-9a18-85d613c77076": "タイプ",
+    "5bcaf853-8e8d-4c0b-b8f8-f25fcc0a16b3": "経費の検索",
+    "638dcad7-7f29-46fc-9258-5195dc6c3a15": "{{format approved_amount \"decimal\"}} {{transaction_currency_code}}",
+    "d57e6fdd-c655-4caa-a6b8-cb62f001da7f": "Report ID",
+    "a7814483-ee11-4391-afb8-df9393c618cd": "取引金額",
+    "f4490947-4501-4d48-a3c6-9b9b2185d515": "ステータス",
+    "6c46135a-3e61-4a9d-980a-abe96b526477": "Cash Advanced",
```
Comment (ghost) [inline bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:69]: Why not translate this string?

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json [hunk 4]
```diff
@@ -1,173 +1,173 @@
...
+    "e408abc3-b922-4ee1-b073-5f07927b9494": "日付",
+    "6a3aae28-856e-4260-9ff5-9977c6d9fda1": "表示するデータがありません",
+    "6dd5904d-632b-4dbe-8e34-6204371a9aeb": "経費を検索して表示します。 ",
+    "e1dfed7f-d2be-4265-8a3a-ff81aa2d296a": "提出日",
+    "36ed4ba9-b76c-4b00-9acd-f4e7c9f0c06b": "領収書",
+    "7eed4e88-40a2-448b-b889-caaa7d43c1c1": "Approved Amount",
```
Comment (ghost) [inline bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:75]: Why not translate this string?

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json [hunk 5]
```diff
@@ -1,173 +1,173 @@
...
+    "7d18b289-bfc8-49eb-9b54-2c2ac8b17784": "経費",
+    "fedc17e1-fa84-44e4-8aec-c19fee060d73": "ベンダーの説明",
+    "480ee02e-15a8-4b74-ae12-b7093633a8c6": "承認者",
+    "d05f8932-7523-4522-b5a1-5bd3aa7c30b2": "Expense Report Details"
```
Comment (ghost) [inline bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:79]: Why not translate this string?

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json [hunk 6]
```diff
@@ -1,173 +1,173 @@
...
+    "c602650c-b691-4374-b44b-ef56eef32fd1": "支出カテゴリ",
+    "44dc30c3-f3f8-437d-abdf-dc4202abd208": "取引金額",
+    "28e10e8e-22d8-4458-99b8-2a7faf343bc4": "提出した経費レポートを検索して表示します。 ",
+    "717f16ce-ede5-4146-9b82-480d4e15495c": "申請の差戻し",
+    "945d0fea-5d3a-4e4e-9345-232ec0eac8c8": "経費ポリシー",
+    "ee1cc89f-7a71-4b7f-bbb7-0cfd1530336b": "申請が承認されました",
+    "23df46cb-8113-4bcf-bcee-82fbd462562e": "ステータス",
+    "38a5df72-9798-42e5-b9eb-418d8705165e": "表示するデータがありません",
+    "81f45b31-9e4a-46ce-9ae3-137858d2f5a7": "自分の保留中の経費レポート",
+    "ca4b06fe-9fd4-41f4-b272-25f5ecfa8df3": "Expense Details",
```
Comment (ghost) [inline bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:102]: Why not translate this string?

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json [hunk 7]
```diff
@@ -1,173 +1,173 @@
...
+    "16216c35-ea37-406c-8333-d01135a08117": "タイプ",
+    "c14873a4-0ac1-4c32-b5e1-bcbaeafb0f14": "{{format total \"decimal\"}} {{currency_code}}",
+    "6702460c-5461-4d9a-8eb3-a150f55d50bc": "取引金額",
+    "308e9f76-f2dc-4cf4-870b-6676c917b99f": "レポートの検索",
+    "813170e4-d5b8-4dff-a3ca-2bd9ec095167": "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
+    "5abaa283-a651-418d-8669-5bf96565d5ba": "提出日",
+    "3316c787-2fd6-42f0-983f-26a84934cb6c": "チェックボックス",
+    "2bdb055d-58df-4264-b163-6a5d80493236": "総額",
+    "f4ea29d7-d2da-4c50-8a4f-e8286c6c6517": "経費の詳細",
+    "6972a08a-a606-457b-a418-4f64036f4d11": "Expense Report Details",
```
Comment (ghost) [inline bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:117]: Why not translate this string?

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json [hunk 8]
```diff
@@ -1,173 +1,173 @@
...
+  "99cfe590-6f49-4342-925c-c8c5e902989e": {
+    "440e9042-6cce-49f0-a88d-e57095d71020": "{{format total \"decimal\"}} {{currency_code}}",
+    "2e6177a8-7eca-426e-a07f-559d1d227de0": "経費の詳細",
+    "72fbc197-667b-4303-83d6-3cabce38ca50": "コメント",
+    "067268a4-44f9-4d01-b0f6-7b8e0d3d2d86": "タイプ",
+    "5911041b-1152-44ae-a882-1e05f8f303fb": "{{owner_name}} submitted a {{format total \"decimal\"}} {{currency_code}} expense report for \"{{name}}\".",
+    "63520869-6193-4192-8e83-c6025d5cee76": "自分の保留中の承認",
+    "d2d0d50b-1ef0-4c7a-aa2e-5b98493f473f": "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
+    "46de609b-f5d0-461b-952a-08c06402a010": "差し戻す",
+    "0505afb5-3153-4d56-b5d8-bc7864d21baf": "Expense Details",
```
Comment (ghost) [inline bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:132]: Why not translate this string?

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json [hunk 9]
```diff
@@ -1,173 +1,173 @@
...
+    "48ad05d7-a220-4e2d-ab9d-cc79b378ece5": "領収書",
+    "a465f98f-69b3-4435-b52d-444b5f6882ca": "日付",
+    "ce3df83f-bd4a-4442-a9d9-7cb9dfa27ff5": "新しい申請",
+    "03370f8e-4937-4fb9-8152-cad2543a1d79": "提出日",
+    "2b8cf1f8-dbb6-474f-ba01-ed224558bd5a": "レポート番号",
+    "b01f78cb-334d-49f9-b2c4-f456589bbf6f": "新しいレポート",
+    "3838ba75-facb-4d62-b237-c3c491205a4e": "Expense Report Details",
```
Comment (ghost) [inline bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:139]: Why not translate this string?

**Review by ghost — APPROVED** [review]
Some strings have not been translated, could you please take a look?

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json (hunk 2 above)
Comment (sivanatase1-ctxs) [reply bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:58]: @yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json (hunk 3 above)
Comment (sivanatase1-ctxs) [reply bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:69]: @yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json (hunk 4 above)
Comment (sivanatase1-ctxs) [reply bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:75]: @yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json (hunk 5 above)
Comment (sivanatase1-ctxs) [reply bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:79]: @yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json (hunk 6 above)
Comment (sivanatase1-ctxs) [reply bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:102]: @yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json (hunk 7 above)
Comment (sivanatase1-ctxs) [reply bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:117]: @yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json (hunk 8 above)
Comment (sivanatase1-ctxs) [reply bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:132]: @yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.

File: bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json (hunk 9 above)
Comment (sivanatase1-ctxs) [reply bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json:139]: @yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.

**Pull request merged**